formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -60,45 +60,45 @@ let x = "hello" [@onHello];
 
 let x = "hello" [@onHello];
 
-let x = "hello" ^ ("goodbye" [@onGoodbye]);
+let x = "hello" ^ "goodbye" [@onGoodbye];
 
-let x = ("hello" [@onHello]) ^ "goodbye";
+let x = "hello" [@onHello] ^ "goodbye";
 
-let x = ("hello" [@onHello]) ^ "goodbye";
+let x = "hello" [@onHello] ^ "goodbye";
 
-let x = "hello" ^ ("goodbye" [@onGoodbye]);
+let x = "hello" ^ "goodbye" [@onGoodbye];
 
-let x = ("hello" ^ "goodbye") [@onEverything];
+let x = (^) "hello" "goodbye" [@onEverything];
 
-let x = 10 + (20 [@on20]);
+let x = 10 + 20 [@on20];
 
-let x = 10 + (20 [@on20]);
+let x = 10 + 20 [@on20];
 
-let x = (10 [@on10]) + 20;
+let x = 10 [@on10] + 20;
 
-let x = (10 [@on10]) + 20;
+let x = 10 [@on10] + 20;
 
-let x = (10 + 20) [@attrEverything];
+let x = (+) 10 20 [@attrEverything];
```

Comment:
To simplify the implementation (and also learning how to use attributes), I just made it so that if you want to apply an attribute to an infix function, you must use the same syntactic patters as with regular function application. Every infix/prefix identifier can be used as a regular identifier, so it works. This simplifies things greatly. We likely had bugs with attributes with various combinations of infix/prefix/unary and now that everything must follow this syntax for attributes, I'm much more confident that everything is correct.